src/routes/docs/tutorials/apple/step-6/+page.markdoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: tutorial
+title: Add authentication
+description: Add authentication to your SwiftUI application.
+step: 6
+---
+
+# Authentication {% #authentication %}
+
+Add the following code to AuthScreen.swift inside **Auth/View** folder. Basicly this UI handle both login and register user.
```

Comment:
Should it be AuthView instead of screen?  What's the convention/difference here？